README.md

```diff
@@ -1,212 +1,217 @@
-
-### Add the root CA that was created to your local truststore.
-
-The certificate can be downloaded from the browser Certificate Viewer by navigating to `https://rekor.<base_domain>`.
-Download the _root_ certificate that issued the Rekor certificate.
-In Red Hat based systems, the following commands will add a CA to the system truststore.
-
-```shell
-$ sudo openssl x509 -in ~/Downloads/root-cert-from-browser -out tas-ca.pem --outform PEM
-$ sudo mv tas-ca.pem /etc/pki/ca-trust/source/anchors/
-$ sudo update-ca-trust
-```
-
-## Signing a Container
+* https://rekor.`<base_hostname>`
+* https://fulcio.`<base_hostname>`
+* https://tsa.`<base_hostname>`
+* https://tuf.`<base_hostname>`
 
-Utilize the following steps to sign a container that has been published to an OCI registry
-
-1. Export the following environment variables substituting `BASE_HOSTNAME`, `KEYCLOAK_URL` and if necessary also `KEYCLOAK_REALM` with the values used as part of the provisioning:
-
-```shell
-export BASE_HOSTNAME="TODO-provide-base-hostname"
-export KEYCLOAK_URL="TODO-your-keycloak-url"
-export KEYCLOAK_REALM=trusted-artifact-signer
-
-export TUF_URL=https://tuf.$BASE_HOSTNAME
-export OIDC_ISSUER_URL=$KEYCLOAK_URL/auth/realms/$KEYCLOAK_REALM
-export COSIGN_FULCIO_URL=https://fulcio.$BASE_HOSTNAME
-export COSIGN_REKOR_URL=https://rekor.$BASE_HOSTNAME
-export COSIGN_MIRROR=$TUF_URL
-export COSIGN_ROOT=$TUF_URL/root.json
-export COSIGN_OIDC_CLIENT_ID=$KEYCLOAK_REALM
-export COSIGN_OIDC_ISSUER=$OIDC_ISSUER_URL
-export COSIGN_CERTIFICATE_OIDC_ISSUER=$OIDC_ISSUER_URL
-export COSIGN_YES="true"
-export SIGSTORE_FULCIO_URL=$COSIGN_FULCIO_URL
-export SIGSTORE_OIDC_ISSUER=$COSIGN_OIDC_ISSUER
-export SIGSTORE_REKOR_URL=$COSIGN_REKOR_URL
-export REKOR_REKOR_SERVER=$COSIGN_REKOR_URL
-```
-
-2. Initialize the TUF roots
-
-```shell
-cosign initialize
-```
-
-Note: If you have used `cosign` previously, you may need to delete the `~/.sigstore` directory
-
-3. Sign the desired container
-
-```shell
-cosign sign -y <image>
-```
-
-Authenticate with the Keycloak instance using the desired credentials.
+## Prerequisites
 
-4. Verify the signed image
+* RHEL x86\_64 9.2 or greater.
+* Command-line access to the Ansible control node with a user that has `sudo` privileges.
+* Installation and configuration of Ansible on a control node to perform the automation.
+* Installation of the Ansible collections on the control node, unless installing from the Ansible Automation Hub.
+  > [!NOTE]
+  To install the required dependencies automatically, you can run the `ansible-galaxy install` command.
+* An OpenID Connect (OIDC) provider, such as [Keycloak](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/).
+* The ability to resolve the ingress host names, by using the Domain Name System (DNS) or the `/etc/hosts` file.
+* Optional.
+  Installation of the `podman` and [`cosign`](https://github.com/sigstore/cosign) binaries to verify that the RHTAS service is working as expected.
```

Comment:
True, if the user already has a signed container image to verify, but what if they don't, and they still want to do a simple verification to see if the RHTAS service is working?  Starting with step 3 in the verification section, I've added a simple verification test.  I have the user create  a very simple container (which is empty), and then I ask them to sign and verify that container image.  In this case we do need them to have `podman`.  I'm going to leave this as-is.

Reply:
Ok, I see what you mean. It's also possible to sign and verify a blob (e.g. a text file) which doesn't require podman at all, but I'm ok if you want to leave this as-is, because signing/verifying container images is likely the most prominent usecase and we want to showcase it.